### PR TITLE
Update gitversion

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: paket.lock [hunk 1]
```diff
@@ -878,4 +878,4 @@ GROUP Build
 NUGET
   remote: https://api.nuget.org/v3/index.json
     FAKE (5.0.0-beta010)
-    GitVersion.CommandLine (3.6.5)
+    GitVersion.CommandLine (4.0.0-beta0012)
```
Comment (tlycken) [inline paket.lock]: Is there something specific we need from the 4.0-pre chain? Why not stick with the stable release?

File: build.fsx [hunk 2]
```diff
@@ -29,48 +29,46 @@ let installDotNet _ = dotnetCliPath <- DotNetCli.InstallDotNetSDK dotnetCliVersi
...
+    let gitVersion = Fake.GitVersionHelper.GitVersion (fun ps -> { ps with ToolPath = gitVersionPath })
+    if Fake.EnvironmentHelper.getEnvironmentVarAsBool "APPVEYOR" then
+        let version = { gitVersion with BuildMetaData = Fake.AppVeyor.AppVeyorEnvironment.BuildNumber }
+        Fake.AppVeyor.UpdateBuildVersion version.InformationalVersion
+        version
+    elif Fake.EnvironmentHelper.getEnvironmentVarAsBool "TRAVIS" then
+        let version = { gitVersion with BuildMetaData = Fake.EnvironmentHelper.environVar "TRAVIS_JOB_NUMBER" }
+        version
+    else
+        { gitVersion with BuildMetaData = "local" }
```
Comment (tlycken) [inline build.fsx]: We can discuss whitespace all day long, and I don't have a strong enough preference to fight for it, but I do think that whitespace changes to big chunks of code like this should be isolated in its own PR, to make review of non-whitespace changes easier.

Comment (tlycken) [reply build.fsx]: As an aside, did you make these changes with some formatting tool? If so, which one, and with what settings?

If there's a standard tool to use, and it's possible to save its settings in a file and check it in, we should do that. Then, every developer can just use the formatting tool before submitting a PR, and the code will automatically be styled the same across the repo.

Comment (forki) [reply build.fsx]: It's common practice in F# to do 4 blanks. I did not plan to merge it here, just needed another diff to kick the CI

File: paket.lock (hunk 1 above)
Comment (forki) [reply paket.lock]: there is GitTools/GitVersion#1043 which suggested the Gitversion error was already fixed. So I tried that. but then I had different error

**Comment by forki** [comment]
now we have "Could not find a 'develop' or 'master' branch, neither locally nor remotely."

**Comment by tlycken** [comment]
https://github.com/GitTools/GitVersion/issues/912 ?

**Comment by forki** [comment]
It's green ;-)
